vector-config/src/main/java/im/vector/app/config/Config.kt

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2022 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package im.vector.app.config
+
+/**
+ * Set of flags to configure the application.
+ */
+object Config {
```

Comment:
:100:  is the plan for this to become our source of truth?

(with some values being pulled in via the `BuildConfig`)

Reply:
Yes, this is what I have in mind. At the end they will be no `BuildConfig` fields anymore. (Also note that this class does not have access to `vector` module's `BuildConfig` class.

So we have:
- `Config` here, in this dedicated super simple configuration module;
- `VectorFeatures` to be able to override values at runtime.